.github/workflows/ci-infra.yml

```diff
@@ -5,11 +5,15 @@ on:
     branches:
       - main
     paths:
+      - bin/**
       - infra/**
+      - .github/workflows/**
       - .github/workflows/ci-infra.yml
   pull_request:
     paths:
+      - bin/**
       - infra/**
+      - .github/workflows/**
       - .github/workflows/ci-infra.yml
```

Comment:
Since you're adding `.github/workflows/**`, are lines 11 and 17 needed? https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

Reply:
yeah good point, they seem redundant now